great_expectations/core/util.py

```diff
@@ -173,6 +188,14 @@ def ensure_json_serializable(data):
     import numpy as np
     import pandas as pd
 
+    try:
+        import pyspark
+    except ImportError:
+        pyspark = None
+        logger.debug(
+            "Unable to load pyspark and pyspark.sql; install optional spark dependency for support."
+        )
+
```

Comment:
Are we having a duplicate of the `import pyspark` section?  Lines 191 -- 198 seem to be the same as lines 66 -- 72...

Reply:
good point. I was following the existing pattern for pandas and numpy imports, but I think they would be much more efficient if it was moved to the top